### PR TITLE
feat: Use Kubernetes Dashboard Helm chart

### PR DESCRIPTION
**Pull request by maciaszczykm**

### CODE REVIEW

File: plural/helm/console/values.yaml [hunk 1]
```diff
@@ -188,80 +188,11 @@ test-base:
...
-    pinnedCRDs: []
-    # - kind: customresourcedefinition
-    #  #  Fully qualified name of a CRD
-    #  name: prometheus.monitoring.coreos.com
-    #  #  Display name
-    #  displayName: Prometheus
-    #  #  Is this CRD namespaced?
-    #  namespaced: true
\ No newline at end of file
+  enabled: true # todo disable before merge
```
Comment (michaeljguarino) [inline plural/helm/console/values.yaml:191]: yes, disable pls

File: plural/helm/console/templates/ingress.yaml [hunk 2]
```diff
@@ -18,6 +18,15 @@ spec:
...
       paths:
+      {{ if .Values.dashboard.enabled }}
+      - path: /api
+        pathType: Prefix
+        backend:
+          service:
+            name: {{ template "kubernetes-dashboard.fullname" (index $.Subcharts "kubernetes-dashboard") }}-{{ .Values.dashboard.api.role }}
+            port:
+              name: {{ .Values.dashboard.api.role }}
+          {{ end }}
```
Comment (michaeljguarino) [inline plural/helm/console/templates/ingress.yaml:29]: is the indentation here goofy?

File: charts/console/Chart.yaml [hunk 3]
```diff
@@ -3,14 +3,19 @@ name: console
...
     version: 0.0.5
     repository: file://../../plural/helm/kas
   - name: controller
     version: 0.0.21
     repository: file://../controller
+  - name: kubernetes-dashboard
+    alias: dashboard
+    version: 7.0.1
+    repository: https://kubernetes.github.io/dashboard
+    condition: kubernetes-dashboard.enabled
```
Comment (michaeljguarino) [inline charts/console/Chart.yaml:18]: shouldn't this be `dashboard.enabled`?

Comment (maciaszczykm) [reply charts/console/Chart.yaml:18]: Yep, it should. Added this before alias and forgot to update.

File: plural/helm/console/templates/ingress.yaml (hunk 2 above)
Comment (maciaszczykm) [reply plural/helm/console/templates/ingress.yaml]: It was, fixed.

**Review by michaeljguarino — APPROVED** [review]
this lgtm but we should definitely disable it by default for now

**Pull request merged**